### DISCUSSION TITLE
yarn构建失败

### DISCUSSION THREAD

**Discussion by yanshifeng** [umijs/umi#11354] [Q&A]
error Couldn't find any versions for "@umijs/utils" that matches "4.0.72"
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Couldn't find any versions for "@umijs/utils" that matches "4.0.72"

**Comment by fz6m** [comment]
删了 node_modules 重装看看，如还有问题，需要给一个最小复现。
